libraries/Parsers/Microsoft.Bot.Builder.Parsers.LU/Parser/ImportSection.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Bot.Builder.Parsers.LU.Parser
+{
+    public class ImportSection : Section
```

Comment:
Each *Section should have xml documentation describing how the actual thing to be parsed looks like as an introduction for future maintainers